src/maestro/cli/commands.py

```diff
@@ -411,6 +411,8 @@ def run(self):
                 duration_ms=duration_ms,
             )
 
+        except KeyError as e:
+            Console.error(f"processing yaml file. key: {str(e)}")
```

Comment:
Should this be:
```suggestion
            Console.error(f"Error processing yaml file. key: {str(e)}")
```
?

Reply:
No, `Console.error` puts the "Error:" in front of message.  So there are 2 "Error"s in the message.